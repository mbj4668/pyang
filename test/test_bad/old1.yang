module old1 {
  namespace urn:old1;
  prefix o1;

  import u {
    prefix x;
    revision-date 2015-11-06; // error, cannot import 1.1 by rev
  }
}
