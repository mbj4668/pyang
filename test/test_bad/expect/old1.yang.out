old1.yang:5: error: BAD_IMPORT_YANG_VERSION
old1.yang:5: warning: UNUSED_IMPORT
u.yang:6: error: BAD_INCLUDE_YANG_VERSION
u.yang:12: error: FEATURE_NOT_FOUND
u.yang:16: error: BAD_VALUE
u.yang:20: error: BAD_VALUE
u.yang:23: error: UNEXPECTED_KEYWORD
u.yang:36: error: TYPE_VALUE
u.yang:55: error: KEY_BAD_SUBSTMT
u.yang:56: error: KEY_BAD_SUBSTMT
u.yang:65: error: IDENTITY_NOT_FOUND
u.yang:71: error: TYPE_VALUE
u.yang:77: error: TYPE_VALUE
u.yang:84: error: DEFAULT_AND_MIN_ELEMENTS
u.yang:91: error: LEAFREF_IDENTIFIER_NOT_FOUND
u.yang:107: error: LEAFREF_IDENTIFIER_NOT_FOUND
u.yang:137: error: TYPE_VALUE
u.yang:143: error: BAD_ENUM_VALUE
u.yang:155: error: BAD_ENUM_VALUE
u.yang:169: error: TYPE_VALUE
u.yang:175: error: BAD_BIT_POSITION
u.yang:187: error: BAD_BIT_POSITION
u.yang:203: error: TYPE_VALUE
u.yang:212: error: TYPE_VALUE
u.yang:222: error: UNEXPECTED_KEYWORD_USES
u.yang:226: error: UNEXPECTED_KEYWORD_USES
